docs/aws.md

```diff
@@ -555,7 +555,7 @@ This client factory has the following configurable catalog properties:
 | client.assume-role.arn            | null, requires user input                | ARN of the role to assume, e.g. arn:aws:iam::123456789:role/myRoleToAssume  |
 | client.assume-role.region         | null, requires user input                | All AWS clients except the STS client will use the given region instead of the default region chain  |
 | client.assume-role.external-id    | null                                     | An optional [external ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html)  |
-| client.assume-role.timeout-sec    | 1 hour                                   | Timeout of each assume role session. At the end of the timeout, a new set of role session credentials will be fetched through a STS client.  |
+| client.assume-role.timeout-sec    | 1 hour                                   | Timeout of each assume role session. At the end of the timeout, a new set of role session credentials will be fetched through an STS client.  |
```

Comment:
I think we can leave this unchanged. "a STS client" is correct

Reply:
Before abbreviations like SEO or UN, write a or an, depending on the sound of the first letter. For example, S is a consonant but pronounced /e/, so you would write "an SEO expert", and U is a vowel but pronounced /j/, so you would write "a UN declaration".

Likewise STS would be pronounced as "ess-tee-ess". In this case S is a consonant but pronounced /e/.


I've included links below that you might find helpful.
https://editorsmanual.com/articles/indefinite-article-a-an-with-abbreviations/

Thank you for your attention to detail.